data/publiccloud/terraform/ec2.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     aws = {
-      version = "= 5.14.0"
+      version = "= 5.58.0"
```

Comment:
That's quiet a version bump. We will require heavy testing for this one and I would recommend to not merge this at the end of the week.

Reply:
This has been discarded.